containers/verifier/install-dependencies.sh.j2

```diff
@@ -1,44 +1,18 @@
 #!/bin/bash -e
 
-readarray dependencies < ${BASH_SOURCE%/*}/pyenv-build-dependencies.txt
-
 dependencies+=(
     # Required by psycopg2 python package (PostgreSQL support)
     libpq-dev
 
     # Update package lists for Debian and install dependencies
     # Packages needed to build concent(python extensions, dependencies of golem-messages in particular)
     libssl-dev
-    git
     pkg-config
-    libffi-dev
-    software-properties-common
-
-    # Packages needed to build Golem's imgverifier image (which our image is based on)
-    libglib2.0-0
-    libsm6
-    libxrender1
-    wget
-    libopenexr-dev
-    xz-utils
-    zlib1g-dev
-)
-
-testing_dependencies=(
-    # Packages needed to build python extensions, dependencies of golem-messages in particular
-    # They need to be installed from testing, because on Debian Jessie they are too old or non-existent
-    binutils
-    build-essential
+    gcc
```

Comment:
What are we using `gcc` for?

Reply:
The same as https://github.com/golemfactory/concent-deployment/pull/361#discussion_r266460697